content/docs/guides/contribute/batch-updating-dependencies.en.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Batch dependency updates"
+linkTitle: "Batch dependency updates"
+weight: 20
+description: ""
+---
+
+
+## For editoast
+
+We use dependabot on the project to signal when dependencies are outdated. We do not use dependabot to automatically update dependencies, as we want to merge all updates at once and review the changes.
+
+Here is the process to update dependencies:
+
+1. Change the versions.
+    * *If you're using VSCode* you can install the `crates` extension and run the "update all dependencies" command.  
```

Comment:
```suggestion
    * *If you're using VSCode* you can install the `serayuzgur.crates` extension and run the "update all dependencies" command.  
```

Reply:
We could also include a ling to https://marketplace.visualstudio.com/items?itemName=serayuzgur.crates?